src/message/html/HtmlInAppLinks.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+import { connect } from 'react-redux';
+import { getNarrowFromLink, getMessageIdFromLink } from '../../utils/url';
+import renderHtmlChildren from './renderHtmlChildren';
+import { getMessages } from '../../api';
+
+class HtmlInAppLinks extends Component {
+
+  render() {
+    const { href, auth, childrenNodes, cascadingStyle, doNarrow, users } = this.props;
+    return (
+      <Text
+        onPress={async () => {
```

Comment:
```js
onPress={someFuncInlinedHere}
```
should always be written as 
```js
onPress={this.handlePress}
...
handlePress = async () =>
```

The first option creates a new function for every single component, while the latter just once.